src/systems/physics-system.js

```diff
@@ -241,6 +243,12 @@ export class PhysicsSystem {
     return bodyId;
   }
 
+  updateRigidBody(eid, options) {
```

Comment:
Yeah, it's a bit unfortunate that we have to go back and forth between bitECS and Ammo to keep everything in sync. As the entry point for the physics life cycle is the physics system, I wonder it would make sense to encapsulate all the component life cycle there and let that system manage it all. Instead of going through the hops of adding a component and have that intermediate `bit-physics-system` to map that to the physics system, just call the physics system and it will add/update the component under the hood and keep everything in sync.

At least this will save some method calls and also remove that bit-physics bridge system.